Resources/Prototypes/_RMC14/Entities/Structures/Base/base_structure.yml

```
@@ -122,3 +122,40 @@
     damageCoefficient: 12.5 # Walls take 15x explosive damage, resin walls reduce that by 15%
   - type: DamageMultiplierFlags
     flags: Turf
+
+# Invisible Wall
+
+- type: entity
+  id: RMCWallInvisibleBulletPassible
+  name: wall blocker
+  suffix: Projectile Passible
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Transform
+    anchored: true
+  - type: Physics
+    bodyType: Static
+  - type: PlacementReplacement
+    key: walls
+  - type: Occluder
+    enabled: false
+  - type: Airtight
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.5,0.5"
+        mask:
+        - FullTileMask
+        layer:
+        - HighImpassable
+        - MidImpassable
+        - LowImpassable
+        - InteractImpassable
+        density: 1000
+  - type: RMCDropshipBlocked
+
+
+
```

Small nitpick, two too many new lines at the end of the file
```suggestion
  - type: RMCDropshipBlocked

```